pkg/metrics/metrics.go

```diff
@@ -191,17 +185,14 @@ func ResetThreeScaleUserAction() {
 	ThreeScaleUserAction.Reset()
 }
 
-func SetNumTenants(numTenants string) {
-	NumTenants.Reset()
-	NumTenants.WithLabelValues(numTenants).Set(float64(1))
-}
-
-func ResetNoTenantRealms() {
-	NoTenantRealm.Reset()
+func SetTotalNumTenants(numTenants int) {
+	//TotalNumTenants.Reset()
```

Comment:
I spotted these resets are commented out - I think you're right and they aren't necessary, perhaps they could just be removed

Reply:
Good catch I forgot to remove those